Domains/4-Application/Connectors/CivilInfrastructureFramework/CifBridge.ecschema.xml

```diff
@@ -1026,7 +1026,7 @@
     <ECEntityClass typeName="CrossFrameHolderAspect">
         <BaseClass>cifcmn:CivilPresentation</BaseClass>
         <ECProperty propertyName="CrossFrameHolder_Name" typeName="string" displayLabel="Name" category="CrossFrameProperties_Cross_Frame_Properties" priority="299999"/>
-        <ECProperty propertyName="CrossFrameHolder_TotalWeight" typeName="double" displayLabel="Total Components Weight" category="CrossFrameProperties_Cross_Frame_Properties" priority="299998" kindOfQuantity="cifu:LENGTH"/>
+        <ECProperty propertyName="CrossFrameHolder_TotalWeight" typeName="double" displayLabel="Total Components Weight" category="CrossFrameProperties_Cross_Frame_Properties" priority="299998" kindOfQuantity="cifu:MASS"/>
```

Comment:
Because the kindOfQuantity is changed for an existing property AND the new kindOfQuantity has a different persistence unit than the old kindOfQuantity, you will also need a custom attribute here for schema update purposes. This ECProperty line should look like this:

    <ECProperty propertyName="CrossFrameHolder_TotalWeight" typeName="double" displayLabel="Total Components Weight" category="CrossFrameProperties_Cross_Frame_Properties" priority="299998" kindOfQuantity="cifu:MASS">
        <ECCustomAttributes>
            <AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
                <From>u:M</From>
                <To>u:KG</To>
            </AllowUnitChange>
        </ECCustomAttributes>
    </ECProperty>